src/pages/projectSettings/tabs/ViewsAndFiltersTab/getFormSchema.ts

```diff
@@ -2,44 +2,15 @@ import { GetFormSchema } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
 import widgets from "components/SpruceForm/Widgets";
 import { ProjectHealthView } from "gql/generated/types";
+import { ProjectType } from "../utils";
 
```

Comment:
What happens if the repo and project filter use the same filter expression? Should they count as separate filters or should they be deduplicated? (For the project filters, it errors if you try to define the same filter expression more than once)

Reply:
Good callout, added repo values to the same validation